src/components/EditorBottomBar.tsx

```diff
@@ -1,15 +1,18 @@
 import "./EditorBottomBar.css";
+import { useIntl } from "react-intl";
 
 interface EditorBottomBarProps {}
 
 export const EditorBottomBar = ({ ...otherProps }: EditorBottomBarProps) => {
+  const intl = useIntl();
+
   return (
     <div className="ed-cta-footer" {...otherProps}>
       <button type="button" className="dp-button button-medium secondary-green">
-        Salir y editar luego
+        {intl.formatMessage({ id: "exit_edit_later" })}
       </button>
       <button type="button" className="dp-button button-medium primary-green">
-        Continuar
+        {intl.formatMessage({ id: "continue" })}
```

Comment:
En estos casos yo usaría el componente en lugar del hook

```suggestion
        <FormattedMessage id="continue" />
```

Reply:
Applied